plugins/woocommerce/client/blocks/assets/js/blocks/product-details/edit.tsx

```diff
@@ -19,7 +19,11 @@ import { ProductDetailsEditProps } from './types';
 const TEMPLATE: InnerBlockTemplate[] = [
 	[
 		'woocommerce/accordion-group',
-		{},
+		{
+			metadata: {
+				isProductDetailsInnerBlock: true,
```

Comment:
We are using `isDescendantOf...` everywhere, so I will update this follow the current convention.